integration/secret/secret_test.go

```diff
@@ -231,6 +231,7 @@ func TestSecretsUpdate(t *testing.T) {
 }
 
 func TestTemplatedSecret(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
```

Comment:
Looks like there's some more to be updated to `== "windows"` (I'm fine with doing a follow up)